### DISCUSSION TITLE
Is Khronos Texture (KTX) similar to stb_image.h?

### DISCUSSION THREAD

**Discussion by yeokaiwei** [KhronosGroup/KTX-Software#508] [Q&A] [answered]
Hi,
I was trying to play around with Sascha Willem's examples and I ran into the KTX header files.

Is KTX an image loader like Sean Barrett's STB?

**Comment by MarkCallow** [comment, marked as answer]
No. KTX is a format specific to images used for texturing in 3d graphics. The software in this KTX-Software repo is for creating and loading files in the KTX format. `stbimage` is a loader for many different image file formats. IIRC it does not support reading any formats that can contain multiple images.

**Comment by yeokaiwei** [comment]
Thank you for the answer.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
